Project.toml

```
@@ -36,17 +35,16 @@ DocStringExtensions = "0.8, 0.9"
 LogDensityProblems = "2"
 LogDensityProblemsAD = "1"
 MCMCChains = "5, 6"
-OrdinaryDiffEq = "6"
+OrdinaryDiffEq = "6.92"
 ProgressMeter = "1"
 Requires = "0.5, 1"
 Setfield = "0.7, 0.8, 1"
-SimpleUnPack = "1.1"
 Statistics = "1.6"
 StatsBase = "0.31, 0.32, 0.33, 0.34"
 StatsFuns = "0.8, 0.9, 1"
-LinearAlgebra = "1.6"
```

It's unnecessary to update stdlib compats in this way. In fact, due to potential issues with test environments the current recommendation is to use
```suggestion
LinearAlgebra = "<0.1, 1"
```